internal/impl/jira/config.go

```diff
@@ -0,0 +1,128 @@
+package jira
+
+import (
+	"errors"
+	"net/http"
+	"net/url"
+
+	"github.com/redpanda-data/connect/v4/internal/impl/jira/helpers/http_helper"
+	"github.com/redpanda-data/connect/v4/internal/impl/jira/helpers/http_metrics"
+
+	"github.com/redpanda-data/benthos/v4/public/service"
+)
+
+type jiraProc struct {
```

Comment:
I like the split to different resources files, I'm confused why this one is called config?

Probably jiraProc  belongs to resources, the rest to processor.